source/includes/substitute-placeholders.rst

```diff
@@ -0,0 +1,4 @@
+.. note::
+
+    You must substitute all values in quotes with angle brackets with
```

Comment:
```suggestion
    You must substitute all text in quotes and angle brackets with
```
Suggestion: "In quotes with angle brackets" sounded a bit off to me. Is this a value in quotes with angle brackets: "hello <>"? Also what is a value? Are all values text in strings?